nncf/torch/dynamic_graph/patch_pytorch.py

```diff
@@ -289,6 +290,9 @@ def patch_torch_jit():
 
     import torch
 
+    if "torchvision" in sys.modules:
+        import torchvision  # noqa
+
```

Comment:
This import will allow to export torchvision models to GraphModules, ok. But what if a person decide to quantize their own model? Or use timm repo? Then nncf should be imported in a particular moment of time. I don't think this is the solution of the torch model wrapping issue
CC: @alexsu52